app/views/donations/_actions.html.erb

```diff
@@ -1,4 +1,8 @@
-<div class="btn-container">
-  <%= render("donations/confirm", donation: donation) %>
-  <%= render("donations/decline", donation: donation) %>
+<div class="donation-actions">
+  <h3 class="section-label">Would you like to change your donation status?</h3>
```

Comment:
Instead of adding the markup to this file, could you put  it before [this line](https://github.com/thoughtbot/freshfoodconnect/blob/1c3b400d3b46ae77aa1db469945fd133243653f2/app/views/donations/edit.html.erb#L14)?

Aslo, what do you think about wrapping it in a check for `donation.pending?`:

```erb
<% unless donation.pending? %>
  <!-- the markup -->
<% end %>
```

That way, it'll only prompt the user to "Change" their decision when they've actually made one